README.md

```diff
@@ -59,8 +59,7 @@ Pavel Kolesnikov [ <mailto:pavel@gooddata.com> / [@koles](http://twitter.com/kol
 
 **Actively developed and maintained by**
 
-- [Tomas Korcak](https://github.com/korczis) [ <mailto:korczis@gmail.com> / [@korczis](http://twitter.com/korczis) ]
-- [Tomas Svarovsky](https://github.com/fluke777) [ <mailto:svarovsky.tomas@gmail.com> / [@fluke777](http://twitter.com/fluke777) ]
```

Comment:
maybe move Korczis and Svara into contributors instead of getting rid of them completely?

Reply:
well their credits are on contribution page and they are not contributors any more, may be we can move them to special thanks? or create new section - wht would be the name?

Reply:
ok I decided to move them just next to other contributors